backend/src/ee/routes/v3/serviceTokenData.ts

```diff
@@ -0,0 +1,39 @@
+import express from "express";
+const router = express.Router();
+import { requireAuth } from "../../../middleware";
+import { AuthMode } from "../../../variables";
+import { serviceTokenDataController } from "../../controllers/v3";
+
+router.get(
```

Comment:
i didn't see this getting used anywhere, do we still need it?

Reply:
I’m not sure I understand which part is not needed - I believe all these new endpoints are needed?